src/libraries/System.Private.CoreLib/src/System/DateOnly.cs

```diff
@@ -19,7 +20,7 @@ namespace System
           ISpanFormattable,
           ISpanParsable<DateOnly>
     {
-        private readonly int _dayNumber;
+        internal readonly int _dayNumber;
```

Comment:
>internal [](http://example.com/codeflow?start=8&length=8)

You don't need to change that. We already have the property `DayNumber` which is public.